engine/language-server/src/test/scala/org/enso/languageserver/websocket/json/LibrariesTest.scala

```diff
@@ -609,6 +715,55 @@ class LibrariesTest extends BaseServerTest {
     }
   }
 
+  "editions/listDefinedComponents" should {
+    "include expected components in the list" in {
+      val client = getInitialisedWsClient()
+      client.send(json"""
+          { "jsonrpc": "2.0",
+            "method": "editions/listDefinedComponents",
+            "id": 0,
+            "params": {
+              "edition": {
+                "type": "CurrentProjectEdition"
+              }
+            }
+          }
+          """)
+
+      client.expectJson(json"""
+          { "jsonrpc": "2.0",
+            "id": 0,
+            "result": {
+              "availableComponents" : [ ]
+            }
+          }
+          """)
+
+      val currentEditionName = buildinfo.Info.currentEdition
+      client.send(json"""
+          { "jsonrpc": "2.0",
+            "method": "editions/listDefinedComponents",
+            "id": 0,
+            "params": {
+              "edition": {
+                "type": "NamedEdition",
+                "editionName": $currentEditionName
+              }
+            }
+          }
+          """)
+
+      client.expectJson(json"""
+          { "jsonrpc": "2.0",
+            "id": 0,
+            "result": {
+              "availableComponents" : [ ]
+            }
+          }
```

Comment:
Maybe you could somehow fake the current edition to have any component groups? Feels like the test is a bit incomplete if only the path leading to 'no results' is tested.